terraform/cp4ba/variables.tf

```diff
@@ -0,0 +1,232 @@
+
+variable "ibmcloud_api_key" {
+  description = "IBM Cloud API key (https://cloud.ibm.com/docs/account?topic=account-userapikey#create_user_key)"
+}
+
+variable "cluster_name_id" {
+  default     = ""
+  description = "Enter your cluster id or name to install the Cloud Pak. Leave blank to provision a new Openshift cluster."
+}
+
+variable "entitled_registry_user_email" {
+  type = string
+  description = "Email address of the user owner of the Entitled Registry Key"
+//  validation {
+//    condition = can(regrex("^No resources found+$", var.entitled_registry_user_email))
+//    error_message = "At least one user must be available in order to proceed. Please refer to the README for the requirements and instructions. The script will now exit!"
+//  }
+}
+
+variable "iaas_classic_api_key" {}
+variable "iaas_classic_username" {}
+variable "ssh_public_key_file" {}
+variable "ssh_private_key_file" {}
+variable "classic_datacenter" {}
+
+variable "config_dir" {
+  default     = "./.kube/config"
+  description = "directory to store the kubeconfig file"
+}
+
+variable "region" {
+  description = "Region where the cluster is created"
+}
+
+variable "resource_group" {
+//  name       = "cloud-pak-sandbox-ibm"
+  description = "Resource group name where the cluster will be hosted."
+}
+
+variable "openshift_version" {
+  default     = "4.6"
+  type        = string
+  description = "Openshift version installed in the cluster"
+}
+//
+//variable "hasEntitlementKey" {
+//  type        = string
+//  description = "Do you have a Cloud Pak for Business Automation Entitlement Registry key (Yes/No, default: No):"
+//}
+
+variable "local_registry_server" {
+  description = "Enter the public image registry or route (e.g., default-route-openshift-image-registry.apps.<hostname>).\nThis is required for docker/podman login validation:"
+}
+
+variable "portworx_is_ready" {
```

Comment:
The user shouldn't be able to set this.  This is derived from the `portworx` module.  See the https://github.com/ibm-hcbt/cloud-pak-sandboxes/blob/master/terraform/cp4data/main.tf, line 104.  So remove from variables.tf file.